pkg/workload/workload.go

```diff
@@ -291,7 +328,10 @@ func totalRequestsFromPodSets(wl *kueue.Workload) []PodSetResources {
 			Name:  ps.Name,
 			Count: count,
 		}
-		setRes.Requests = resources.NewRequests(limitrange.TotalRequests(&ps.Template.Spec))
+		specRequests := limitrange.TotalRequests(&ps.Template.Spec)
+		filteredRequests := dropExcludedResources(specRequests, info.excludedResourcePrefixes)
+		effectiveRequests := applyResourceTransformations(filteredRequests, info.resourceTransformations)
```

Comment:
```suggestion
		if features.Enabled(features.ConfigurableResourceTransformations) {
			effectiveRequests := applyResourceTransformations(filteredRequests, info.resourceTransformations)
		}
```

or

```suggestion
		if info.resourceTransformations != nil {
			effectiveRequests := applyResourceTransformations(filteredRequests, info.resourceTransformations)
		}
```

Could we verify if the transformations are enabled? I think that this could often happen since this feature is disabled by default, and then let's verify if this transformation feature never affect on the existing mechanism.

Reply:
done; made it depend on the feature gate.